subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -2051,20 +2051,17 @@ uint16_t ull_conn_lll_max_tx_octets_get(struct lll_conn *lll)
 
 /**
  * @brief Initialize pdu_data members that are read only in lower link layer.
+ *        Fields that are modified after static initialization are only
+ *        re-initialized.
  *
  * @param pdu_tx Pointer to pdu_data object to be initialized
  */
 void ull_pdu_data_init(struct pdu_data *pdu_tx)
 {
-	LL_ASSERT(pdu_tx);
-
-	pdu_tx->cp = false;
-	pdu_tx->rfu = 0U;
-#if !defined(CONFIG_SOC_OPENISA_RV32M1_RISCV32)
-#if !defined(CONFIG_BT_CTLR_DATA_LENGTH_CLEAR)
+#if defined(CONFIG_BT_CTLR_DF_CONN_CTE_TX)
+	pdu_tx->cp = 0U;
```

Comment:
What this was changed from `pdu_tx->cp = false`? This is a boolean flag stored in single bit.